src/openforms/submissions/templates/report/submission_report.html

```diff
@@ -55,13 +55,18 @@ <h2 class="subtitle">{{ submission_step_node.render }}</h2>
                                     {% if component_node.spans_full_width %}
                                         {{ component_node.display_value }}
                                     {% else %}
-                                        <div class="submission-step-row__label">
-                                            {{ component_node.label }}
-                                        </div>
+                                        {% if component_node.component.type == "fieldset" %}
+                                            <span class="submission-step-row__fieldset__label">{{ component_node.label }}</span>
```

Comment:
```suggestion
                                            <span class="submission-step-row__fieldset-label">{{ component_node.label }}</span>
```
within a single class name in BEM, you can only have one occurrence of `__` and one of `--`, so valid is:

* `block`
* `block__element`
* `block__element--modifier`

Invalid is:
* `block__element1__element2`
* `block--modifier1--modifier2`